dbms/src/Flash/Statistics/JoinImpl.cpp

```diff
@@ -20,16 +20,18 @@ namespace DB
 void JoinStatistics::appendExtraJson(FmtBuffer & fmt_buffer) const
 {
     fmt_buffer.fmtAppend(
-        R"("peak_build_bytes_usage":{},"build_side_child":"{}","is_spill_enabled":{},"is_spilled":{})"
-        R"("join_build_inbound_rows":{},"join_build_inbound_blocks":{},"join_build_inbound_bytes":{},"join_build_inbound_allocated_bytes":{}, "join_build_execution_time_ns":{})",
+        R"("peak_build_bytes_usage":{},"build_side_child":"{}","is_spill_enabled":"{}","is_spilled":"{}",)"
+        R"("join_build_inbound_rows":{},"join_build_inbound_blocks":{},"join_build_inbound_bytes":{},)"
+        R"("join_build_inbound_allocated_bytes":{},"join_build_concurrency":{},"join_build_execution_time_ns":{})",
         peak_build_bytes_usage,
         build_side_child,
-        is_spill_enabled,
-        is_spilled,
+        is_spill_enabled ? "true" : "false",
+        is_spilled ? "true" : "false",
```

Comment:
why we need this? won't the fmtlib turn it into "true"/"false" for boolean values?

Reply:
It's really unnecessary